src/logs/exporters/logExporter.ts

```diff
@@ -78,9 +83,13 @@ export class LogExporter {
                     }, this._batchSendRetryIntervalMs);
                     this._retryTimer.unref();
                 }
+                this._statsbeatMetrics?.countSuccess(duration);
```

Comment:
It doesn't look like you actually instantiate _statsbeatMetrics anywhere, please unit tests for this as well